instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteRefreshTest.kt

```diff
@@ -108,7 +160,7 @@ class RouteRefreshTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.ja
 
         assertEquals(sanityDuration, initialDuration, 0.0)
         assertEquals(227.918, initialDuration, 0.0001)
-        assertEquals(230.651, refreshedDuration, 0.0001)
+        assertEquals(1180.651, refreshedDuration, 0.0001)
```

Comment:
should we also check duration from route progress to make sure that NN got actual data?

Reply:
We do that above.

Reply:
yeah, actually If we want to cover as many cases and catch potential regression, we should compare all possible change data from NN

Reply:
That should be tested on the Nav Native side. I'm just checking that roundtripping works.

Reply:
> We do that above.

As far as I see we check durations from legs which are set by Android SDK, not NN 🤔

Reply:
We check the duration from route progress in https://github.com/mapbox/mapbox-navigation-android/pull/5755/files#diff-bef8d61571a3abff3c6dfcd44a23faa32209dcd71521bdd29e50bd3d13fc7e95R146.